src/preload.py

```diff
@@ -389,6 +377,38 @@ def list_images(image):
     return _list_images(image)
 
 
+def get_docker_init_file_content(image):
+    with mount_context_manager(image, 2) as mountpoint:
+        path = os.path.join(
+            mountpoint,
+            'lib',
+            'systemd',
+            'system',
+            'docker.service',
```

Comment:
@pimterry this is definitely only meant for resinOS images.
One of the first things preload does is to find a config.json or device-type.json file.
Before this PR the script was looking for /etc/os-release.
It also assumes that the root partition in the 2nd and that the docker data is on the 6th partition.